### DISCUSSION TITLE
Add a stylesheet feature to the sitemap integration

### DISCUSSION THREAD

**Discussion by sarajw** [withastro/roadmap#668] [Proposal]
"Would love to add a stylesheet feature to the sitemap integration, similar to the [styling features available for the RSS feed](https://docs.astro.build/en/guides/rss/#adding-a-stylesheet). Being able to pass in a custom XSL stylesheet with maybe a sensible default would be tremendous!"

_Originally posted by @johneatmon in https://github.com/withastro/roadmap/discussions/146#discussioncomment-2502586_

Sara: What he said!

I'm currently manually copying and pasting to make this happen, but that means the sitemap I will link to won't update on every build.

**Comment by florian-lefebvre** [comment]
This is now possible I believe: https://docs.astro.build/en/guides/integrations-guide/sitemap/#xslurl
